apps/aecore/src/aec_conductor.erl

```diff
@@ -481,8 +481,10 @@ maybe_publish_block(block_received,_Block) ->
     %% changes the top.
     ok;
 maybe_publish_block(block_created = T, Block) ->
+    aec_events:publish(T, Block),
     %% This is a block we created ourselves. Always publish.
-    aec_events:publish(T, Block).
+    {ok, BlockHash} = aec_blocks:hash_internal_representation(Block),
```

Comment:
Since we are only using block_to_publish in one place and there we use the full block it might be better to just include the block in the message?

Reply:
ok

Reply:
Addressed.